### PR TITLE
DG-1432 Auth Policies for MQ related Items

### PR DESCRIPTION
**Pull request by arpit-at**
## Change description

> Description here

## Type of change
- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: addons/policies/bootstrap_entity_policies.json [hunk 1]
```diff
@@ -3108,6 +3108,190 @@
...
                 ]
             }
+        },
+
+        {
+            "typeName": "AuthPolicy",
+            "attributes": {
+                "name": "CUD_BUSINESS_POLICY",
+                "qualifiedName": "CUD_BUSINESS_POLICY",
+                "policyCategory": "bootstrap",
```
Comment (nikhilbonte21) [inline addons/policies/bootstrap_entity_policies.json:3118]: Just for confirmation, are there any relationships defined between any of these new types, if yes i think we need to bootstrap relationship policies as well

**Review by nikhilbonte21 — CHANGES_REQUESTED** [review]

**Review by nikhilbonte21 — APPROVED** [review]

File: addons/policies/bootstrap_relationship_policies.json [hunk 2]
```diff
@@ -760,6 +760,82 @@
...
+                "policyResourceCategory": "RELATIONSHIP",
+                "policyResources":
+                [
+                    "end-one-entity-classification:*",
+                    "end-two-entity-classification:*",
+                    "end-one-entity:*",
+                    "end-two-entity:*",
+                    "end-one-entity-type:BusinessPolicy",
+                    "end-two-entity-type:BusinessPolicy",
+                    "relationship-type:*"
```
Comment (mehtaanshul) [inline addons/policies/bootstrap_relationship_policies.json:792]: specify relationship-type

File: addons/policies/bootstrap_relationship_policies.json [hunk 3]
```diff
@@ -760,6 +760,82 @@
...
+                "policyResourceCategory": "RELATIONSHIP",
+                "policyResources":
+                [
+                    "end-one-entity-classification:*",
+                    "end-two-entity-classification:*",
+                    "end-one-entity:*",
+                    "end-two-entity:*",
+                    "end-one-entity-type:BusinessPolicy",
+                    "end-two-entity-type:BusinessPolicyException",
+                    "relationship-type:*"
```
Comment (mehtaanshul) [inline addons/policies/bootstrap_relationship_policies.json:830]: specify relationship-type

File: addons/policies/bootstrap_entity_policies.json [hunk 4]
```diff
@@ -3108,6 +3108,190 @@
...
+                "policyUsers": [],
+                "policyGroups": [],
+                "policyRoles": [
+                    "$admin",
+                    "$api-token-default-access"
+                ],
+                "policyResourceCategory": "ENTITY",
+                "policyResources": [
+                    "entity-type:Incident",
+                    "entity-type:BusinessPolicyIncident",
```
Comment (mehtaanshul) [inline addons/policies/bootstrap_entity_policies.json:3193]: Since BusinessPolicyIncident is subtype of Incident, we don't need specify it separately

**Review by nikhilbonte21 — APPROVED** [review]

**Review by mehtaanshul — APPROVED** [review]

**Pull request merged**